### ISSUE TITLE
Lint artifact

### ISSUE THREAD

**Issue by ZacSweers** [JakeWharton/nopen#5]
Open to a PR with a lint analogue of this? Benefit being that it could show up in the IDE as they're typing

**Comment by vanniktech** [comment]
\+ Kotlin

**Comment by ZacSweers** [comment]
I'd actually exclude the lint from running on kotlin code, since it has `open` and I figure this is basically just an implementation of that pattern for Java

**Comment by JakeWharton** [comment]
Yeah I wrote it as an EP check because it only makes sense for Java and no one uses lint outside of Android despite being available.
